### PR TITLE
[libsrt] Update to 1.5.0

### PR DESCRIPTION
**Pull request by fran6co**
**Describe the pull request**

- #### What does your PR fix?
  Add SRT port

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  `all`, `Yes`

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  `Yes`

### CODE REVIEW

File: ports/srt/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+        OUT_SOURCE_PATH SOURCE_PATH
+        REPO Haivision/srt
+        REF v1.5.0
+        SHA512 68ab5fe316cfbbbba31b5b2354f657e23c90e14674f8dac01df2bf98c4776a7fafcd690a4dfad3a340e6be577a22360ca04ef2397c8a0dd507adebdd54dc22fb
+        HEAD_REF master
+        PATCHES
+            000-fix-install-dirs.patch
```
Comment (LilyWangLL) [inline ports/srt/portfile.cmake:8]:
```suggestion
    OUT_SOURCE_PATH SOURCE_PATH
    REPO Haivision/srt
    REF v1.5.0
    SHA512 68ab5fe316cfbbbba31b5b2354f657e23c90e14674f8dac01df2bf98c4776a7fafcd690a4dfad3a340e6be577a22360ca04ef2397c8a0dd507adebdd54dc22fb
    HEAD_REF master
    PATCHES
        000-fix-install-dirs.patch
```

File: ports/srt/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" KEYSTONE_BUILD_STATIC)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" KEYSTONE_BUILD_SHARED)
+
+vcpkg_cmake_configure(
+        SOURCE_PATH ${SOURCE_PATH}
+        OPTIONS
+            -DENABLE_APPS=OFF
+            -DENABLE_CXX11=ON
+            -DENABLE_STATIC=${KEYSTONE_BUILD_STATIC}
+            -DENABLE_SHARED=${KEYSTONE_BUILD_SHARED}
```
Comment (LilyWangLL) [inline ports/srt/portfile.cmake:20]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
        -DENABLE_APPS=OFF
        -DENABLE_CXX11=ON
        -DENABLE_STATIC=${KEYSTONE_BUILD_STATIC}
        -DENABLE_SHARED=${KEYSTONE_BUILD_SHARED}
```

File: ports/srt/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+)
+vcpkg_cmake_install()
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/srt RENAME copyright)
```
Comment (LilyWangLL) [inline ports/srt/portfile.cmake:23]:
```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by LilyWangLL** [comment]
Thanks for your PR. There are some suggestions about format in **portfile.cmake**.
Under normal circumstances we would not accept new libraries with such short names, is it possible to use `libsrt` instead of `srt`?

**Review by LilyWangLL — DISMISSED** [review]

**Review by BillyONeal — APPROVED** [review]
I pushed a nitpick fix and will merge when CI is green

**Pull request merged**